app/styles/partials/footer.scss

```diff
@@ -2,6 +2,9 @@
   display: flex;
   min-height: 100vh;
   flex-direction: column;
+  .main-container {
```

Comment:
"We want this fix only for smaller screen sizes." But larger screen is having the problem too. Okay I will take care of all the screen sizes.